usr/share/nxdk/prelude.inl

```diff
@@ -0,0 +1,38 @@
+set -u
+set -e
+
```

Comment:
Consider some conditional logging like `echo "$0 $@" >> /tmp/$(basename $0).log`

This logging should be conditional and improved, but it should probably be part of these wrappers to ease debugging of complicated build-systems